### DISCUSSION TITLE
[REQUEST] Web UI Dashboard To Show Backups

### DISCUSSION THREAD

**Discussion by encryptblockr** [offen/docker-volume-backup#468] [Ideas]
Any plans for a web UI dashboard to view backups and locations?

For any backup solution a web UI is very important else it will be a wilderness of imaginations

**Comment by m90** [comment]
I'd think including a full blown UI is out of scope for this tool.

However, I'd still be in favor for adding some sort of metrics interface as already discussed in #64. Users that want a graphical interface to these metrics could then set up 3rd party tools like Grafana or similar that provide a dashboard with relevant statistics based on the metrics scraped from the backup container.

**Comment by Persil090** [comment]
> ptblockr

That would be really awesome to have. Can it be reconsidered for the future or is really too complex?

**Reply by m90** [reply]
It's not too complex, but it's out of scope. As already said this tool should definitely be able to emit metrics, but displaying them should be done by other tools.
